guides/user/plugins/user-guide.md

```diff
@@ -114,122 +88,99 @@ This guide uses the [file](https://raw.githubusercontent.com/spinnaker-plugin-ex
 
-# Additional plugin repository commands
+After you have added your plugin repository, you can add your plugin to Spinnaker. The Halyard [command](/reference/halyard/commands/#hal-plugins-add) is:
 
-## List configured plugin repositories
-
-```
-hal plugins repository list
+```bash
+hal plugins add <unique-plugin-id> --extensions=<extension-name> \
+--version=<version> --enabled=true
 ```
 
-## Edit a plugin repository
-
-You can update a repository's URL using `hal`. For example:
+The plugin distributor should provide you with the `unique-plugin-id`, `extensions`, and `version` values as well as any plugin configuration details. If you have to hunt for these values, you can find `unique-plugin-id` and `version` in the `plugins.json` file, but you have to look at the code to find the value for `extensions`. Search for the `@ExtensionConfiguration` annotation, which defines the extension name.
 
-```
-hal plugins repository edit spinnaker-plugin-examples \
-    --url=https://github.com/aimeeu/examplePluginRepository/blob/master/plugins.json
-```
+The following example is from the [pf4jStagePlugin](https://github.com/spinnaker-plugin-examples/pf4jStagePlugin/blob/master/random-wait-orca/src/main/kotlin/io/armory/plugin/stage/wait/random/RandomWaitConfig.kt).
 
-## Delete a plugin repository
+```java
+package io.armory.plugin.stage.wait.random
 
-You can use `hal` to delete a plugin repository. For example:  
+import com.netflix.spinnaker.kork.plugins.api.ExtensionConfiguration
 
+/**
+ * Data in this class maps to the plugin configuration in a service's config YAML.
+ * The data can be key/value pairs or an entire configuration tree.
+ *
+ */
+@ExtensionConfiguration("armory.randomWaitStage")
+data class RandomWaitConfig(var defaultMaxWaitTime: Int)
 ```
-hal plugins repository delete spinnaker-plugin-examples
-```
-
-# Install the plugin
 
-After you add your plugin repository, you must configure Spinnaker to use your plugin.
+You can also list, edit, and delete plugins. See the Halyard [commands](https://spinnaker.io/reference/halyard/commands/#hal-plugins) for a complete list.
 
-This guide uses the [pf4jStagePlugin](https://github.com/spinnaker-plugin-examples/pf4jStagePlugin) plugin as an example.
+Note: `hal plugins enable` and `hal plugins disable` enable or disable all plugins, so use with caution.
 
-## Configure Spinnaker for the plugin
+### Configure the plugin
 
-Use `hal` to enable your plugin so that Spinnaker will load it. Do not use `hal plugins enable`, which enables all plugins.
-
-```
-hal plugins add Armory.RandomWaitPlugin \
-	--enabled=true \
-	--extensions=armory.randomWaitStage \
-    --ui-resource-location=<location-of-plugin-ui-resource> \
-    --version=<version>             
-```
+Halyard does not support configuring plugins, so you should manually edit the `.hal/config` file to configure custom values. Find the `plugins` section, locate your plugin's definition, and then change values in the `config` section.
```

Comment:
Would an example fragment from the halconfig be a good thing to include after this paragraph?

Reply:
Yes of course - had it in, lost it in refactoring!